contrib/terraform-testing-infrastructure/README.md

```diff
@@ -0,0 +1,227 @@
+# Accumulo Testing Infrastructure
+
+## Description
+
+This Git repository contains several [Terraform](https://www.terraform.io/) configurations.
+
+  - `shared_state` creates an AWS S3 Bucket and DynamoDB table that are a prerequisite for the Terraform configuration in `aws`.
+  - `aws` creates the following AWS resources:
+    1. Creates an EFS filesystem in AWS (think NFS). On this filesystem we are going to install Maven, ZooKeeper, Hadoop, Accumulo, and Accumulo Testing
+    2. Creates IP addresses for the EFS filesystem in each AWS availability zone that we use. The IP address allows us to mount the EFS filesystem to EC2 nodes.
+    3. Creates one or more EC2 nodes for running the different components. Currently the configuration uses the m5.2xlarge instance type which provides 8 vCPUs, 32GB RAM, and an EBS backed root volume.
+    4. Runs commands on the EC2 nodes after they are started (5 minutes according to the docs) to install software and configure them.
+    5. Creates DNS A records for the EC2 nodes.
+
+## Prerequisites
+
+You will need to download and install the correct Terraform [CLI](https://www.terraform.io/downloads) for your platform. Put the `terraform` binary on your PATH.
+
+## Shared State
+
+The `shared_state` directory contains a Terraform configuration for creating an S3 bucket and DynamoDB table. These objects only need to be created once and are used for sharing the Terraform state with a team. To read more about this see [remote state](https://www.terraform.io/docs/language/state/remote.html). I used [these](https://blog.gruntwork.io/how-to-manage-terraform-state-28f5697e68fa) instructions for creating these objects. This configuration is simple, there are no variables, but you may need to change the aws region in `main.tf`. If you change the name of the S3 bucket or DynamoDB table, then you will need to update the `main.tf` file in the `aws` directory. To create the S3 bucket and DynamoDB table, run `terraform init`, followed by `terraform apply`.
+
+## Test Cluster
+
+The `aws` directory contains a Terrform configuration and related files for creating an Accumulo cluster on AWS infrastructure. This configuration consists of the following items in this directory:
+
+  - main.tf - the Terraform configuration file
+  - variables.tf - the declaration and default values for variables
+  - templates/ - template files that are processed during the `apply` operation
+  - files/ - non-template files that are uploaded to the cluster during the `apply` operation
+  - conf/ - a non-git tracked directory that contains the rendered template files that are uploaded to the cluster
```

Comment:
I do not see `aws/conf/`. Will it be generated during setup?

Reply:
yes, it's not tracked by git.